sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsOptions.cs

```diff
@@ -76,7 +76,8 @@ string IOptionsFormatter.Format()
         {
             JObject options = new JObject
             {
-                { nameof(MaxDegreeOfParallelism), MaxDegreeOfParallelism }
+                { nameof(MaxDegreeOfParallelism), MaxDegreeOfParallelism },
+                { nameof(MaxDequeueCount), MaxDequeueCount }
```

Comment:
As part of these changes, I think we should fix up the xml comments on the MaxDequeueCount property added in the previous commit above. I see the comments were copied as is from QueuesOptions.

Perhaps a comment like: "Gets or sets the number of times to try processing a blob before giving up and creating a poison queue message for it." And in a remarks section we could detail "Behind the scenes an infrastructure Azure Queue is used as a work queue for discovered blobs that are waiting to be processed. This property therefore governs the maximum number of times we'll attempt to dequeue and process each of those work items."